apps/web/pages/auth/setup/steps/ChooseLicense.tsx

```diff
@@ -0,0 +1,70 @@
+import { DeploymentLicenseType } from "@prisma/client";
+import classNames from "classnames";
+import { useRouter } from "next/router";
+import { Dispatch, SetStateAction } from "react";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc/react";
+
+const ChooseLicense = (props: {
+  isFreeLicense: boolean;
+  setIsLoading: Dispatch<SetStateAction<boolean>>;
+  setIsFreeLicense: Dispatch<SetStateAction<boolean>>;
+}) => {
+  const { t } = useLocale();
+  const router = useRouter();
+  const { isFreeLicense, setIsFreeLicense } = props;
+  const mutation = trpc.viewer.auth.deploymentSetup.useMutation({
+    onSuccess: () => {
+      router.replace(`/auth/setup?step=3${isFreeLicense ? "&category=calendar" : ""}`);
+    },
+  });
+  return (
+    <form
+      id="wizard-step-2"
+      name="wizard-step-2"
+      className="space-y-4"
+      onSubmit={(e) => {
+        e.preventDefault();
+        props.setIsLoading(true);
+        mutation.mutate({
+          licenseType: isFreeLicense ? DeploymentLicenseType.FREE : DeploymentLicenseType.EE,
+        });
+      }}>
+      <div className="grid grid-cols-2 gap-4">
+        <div
+          className={classNames(
+            "cursor-pointer space-y-2 rounded-md border p-4 hover:border-black",
+            isFreeLicense && "border-black"
+          )}
+          onClick={() => setIsFreeLicense(true)}>
```

Comment:
Can we make this a button please

Clickable divs have no a11y support and require you to manually setup things like roles aria label and tab index aswell as a bunch of other things

Reply:
Also these appear to very simular to radio buttons under the hood - Use Radix if you need to do some fancy styling